packages/experiments/src/components/PersonaCoin/PersonaCoin.types.ts

```diff
@@ -92,6 +98,15 @@ export interface IPersonaCoinProps
    * @defaultvalue 'white'
    */
   initialsColor?: string;
+
+  /**
+   * Optional callback used to set a ref to the component
+   */
+  componentRef?: IRefObject<IPersonaCoinViewProps>;
+}
+
+export interface IPersonaCoinViewProps extends IPersonaCoinProps {
+  isPictureLoaded?: boolean;
```

Comment:
From taking a look at the other components, I think the standard we're going for with `ViewProps` is for them to be a `type` instead of an `interface`. So it would be:

```ts
export type IPersonaCoinViewProps = IPersonaCoinProps & {
     isPictureLoaded?: boolean;
}
```

Reply:
It was previously `type` because of the complex mix of picks and such. We should prefer `interface` in general because it's generally more flexible than types. I'll make sure the templates are updated.

Reply:
That's fine with me, i can change that :)

Serious question in that case would be why still prefix with this weird `I` for _interface_ :P

Reply:
Ah I see we do that more often too (a couple lines down) :D

Reply:
Well, Jason just said that we prefer interface, so I guess you were on the right path since the beginning :)

Reply:
Please don't change it, it's fine as-is. :)

In general Fabric has prefixed all types and interfaces with an `I` to distinguish from classes.

Reply:
Too late, but reverted :P